README.md

```diff
@@ -19,7 +93,7 @@ Because this application gives clients full access to the server, it is highly r
 Running with [Docker Compose](https://docs.docker.com/compose):
 
     $ docker-compose up -d
-    $ curl http://localhost:5000/
+    $ curl http://localhost:7575/
```

Comment:
Why was the default port changed?

Reply:
On mac Port 5000 is reserved for Apple AirPlay Receiver.

So when trying to run it, Shaas immediately complained "Address already in use - bind(2) (Errno::EADDRINUSE)"

I randomly changed it to 7575, since that port is not a commonly used port.